flask/avogadro/src/server.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from flask import Flask, request
+
+import avogadro_api as avogadro
+
+app = Flask(__name__)
+
+
+@app.route('/calculate', methods=['POST'])
```

Comment:
We probably want to be a little more explicit with the name `/calculate-mo`?

Reply:
Fixed.